### PR TITLE
Add fedora build

### PR DESCRIPTION
**Pull request by 1efty**
# Description

This adds a build specific for Fedora. I am leaving this as a draft until I'm 100% sure this thing works. 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

The packer build along with the post-build tests will be used to test this. 

- [x] Packer build
- [x] test.sh

### CODE REVIEW

**Comment by 1efty** [comment]
@incredincomp `make build/fedora` is working along with the test-case outlined in the README. We should talk about next steps.

**Comment by incredincomp** [comment]
What do you have in mind bud?

**Comment by 1efty** [comment]
Well

1. Do you want to test?
2. I've also updated the mechanism to find `chromiumPath`

**Comment by 1efty** [comment]
It's missing zap, let me add that really quick...

**Comment by incredincomp** [comment]
im still trying to fix ubuntu go build rn.... but ill give it a go later. Zap cross platform should work, you should be able to do similar to what this branch does too with wget and unzip

**Comment by incredincomp** [comment]
then run it from cmd line as -daemon and you should be able to curl the api no prob

File: the_hunting.sh [hunk 1]
```diff
@@ -54,8 +54,7 @@ httprobeThreads=50
 
 ssh_file="~/.ssh/id_rsa"
 # Good call @1efty
-CHROMIUM="${CHROMIUM:-"chromium"}"
-chromiumPath="$(which $CHROMIUM)"
+cromiumPath="$(which chromium 2>/dev/null || which chromium-browser)"
```
Comment (incredincomp) [inline the_hunting.sh:57]: Needs an ‘h’

Comment (1efty) [reply the_hunting.sh]: oof.

**Comment by 1efty** [comment]
After that `h`, test case runs fine. However, I feel like I am not testing a few things (slack & S3).

**Review by incredincomp — APPROVED** [review]
🎉

**Comment by incredincomp** [comment]
Eek, we’re you ready for that? We can roll that back... I’m on mobile and this is aids lol

**Pull request merged**